solr/core/src/java/org/apache/solr/search/FunctionRangeQuery.java

```diff
@@ -39,10 +40,20 @@ public FunctionRangeQuery(ValueSourceRangeFilter filter) {
     this.cost = 100; // default behavior should be PostFiltering
   }
 
+  @Override
```

Comment:
I'm not sure why changes are here but I suspect it's related to the increased scope.  Maybe this can be punted for another issue.

Reply:
Yea, I really can't remember why that's there, maybe some debug code. I have cleaned it up though. Thanks for catching that!